cmd/export.go

```diff
@@ -33,12 +38,25 @@ func setupExportCommand() *cobraext.Command {
 	exportDashboardCmd.Flags().StringSliceP(cobraext.DashboardIDsFlagName, "d", nil, cobraext.DashboardIDsFlagDescription)
 	exportDashboardCmd.Flags().Bool(cobraext.TLSSkipVerifyFlagName, false, cobraext.TLSSkipVerifyFlagDescription)
 
+	exportInstalledObjectsCmd := &cobra.Command{
+		Use:   "installed-objects",
+		Short: "Export installed Elasticsearch objects",
+		Long:  exportInstalledObjectsLongDescription,
+		RunE:  exportInstalledObjectsCmd,
+	}
+	exportInstalledObjectsCmd.Flags().StringP(cobraext.ExportPackageFlagName, "p", "", cobraext.ExportPackageFlagDescription)
+	exportInstalledObjectsCmd.MarkFlagRequired(cobraext.ExportPackageFlagName)
+	exportInstalledObjectsCmd.Flags().StringP(cobraext.ExportOutputFlagName, "o", "output", cobraext.ExportOutputFlagDescription)
```

Comment:
I'm wondering if we can map it on the existing package structure on disk instead of `output` directory, similarly to other `export`'s subcommand.

Reply:
Unlike dashboards, these objects are not going to be part of a package, so there would be little reason to do this. This, along with other differences may be a reason to don't put this new subcommand under `export`, but not sure where to put it then.

Reply:
Moving the subcommand to a new `dump` command.